webpack/components/NewTemplateSync/components/NewTemplateSyncForm/NewTemplateSyncForm.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { change } from 'redux-form';
```

Comment:
In your opinion, can you finalize the work on the `formik` infrastructure and use it here once it is ready?
Is it going to drastically change the design?

What about refactoring into multiple PRs so we can gradually insert code which is ready?
- You can decide you only use it in the storybook.
- You can create a secret/experimental route (e.g. /ex/templates) and gradually deliver features to the new route until it is ready to replace the current route.